### PR TITLE
Fix crosscompile to raspberry pi3 (arm ca7)

### PR DESCRIPTION
**Pull request by cschlote**
size_t has only 32 bit on the ARM platform. So we need some explicit
casts at the right place. Changed _currInd to be size_t, which matches
best here as it needs the least number of conversions later.

### CODE REVIEW

File: examples/rcv1.d [hunk 1]
```diff
@@ -122,9 +122,9 @@ class RCV1Reader : DataFileReader!(Obs) {
         auto lab_end = countUntil(buff, "  ");
         if(lab_end == -1)
             return false;
-        auto label = labels[to_long(buff[0..lab_end])];
+        auto label = labels[buff[0..lab_end].to!size_t];
```
Comment (rostyboost) [inline examples/rcv1.d]: we should probably do something like `to_long(buff[0..lab_end]).to!size_t` to keep the fast (but unsafe) parsing through [to_long](https://github.com/Netflix/vectorflow/blob/master/src/vectorflow/utils.d#L45)

**Review by rostyboost — CHANGES_REQUESTED** [review]
Thanks for your contribution @cschlote ! Minor comment, otherwise LGTM.

File: examples/rcv1.d (hunk 1 above)
Comment (cschlote) [reply examples/rcv1.d]: Yes.

**Comment by cschlote** [comment]
Added the change.

**Pull request merged**